README.md

```diff
@@ -44,28 +51,55 @@ command line like this:
 ./normalizer < sample.csv > output.csv
 ```
 
+### Web Browser (If you want us to run your code from a browser):
+
+Please write a static webpage that normalizes and displays the data contained in
```

Comment:
One big challenge with only displaying the data is that we have a [standardized tool for comparing the work sample output](https://github.com/trussworks/wsdiff) and that won't work as well if the data is only displayed without a standardized output format.

I was able to find [file-saver](https://www.npmjs.com/package/file-saver) which seems perfect for this and maybe we'd want to consider mentioning it?